src/ray/core_worker/core_worker.cc

```diff
@@ -179,6 +157,18 @@ CoreWorker::CoreWorker(const WorkerType worker_type, const Language language,
       new CoreWorkerDirectActorTaskSubmitter(
           io_service_, std::unique_ptr<CoreWorkerMemoryStoreProvider>(
                            new CoreWorkerMemoryStoreProvider(memory_store_))));
+  auto create_actor_callback = [this](const ActorID &actor_id,
+                                      const TaskSpecification &actor_creation_spec,
+                                      uint64_t num_lifetimes) {
+    if (!raylet_client_->SubmitTask(actor_creation_spec, num_lifetimes).ok()) {
+      // TODO(swang): Mark the actor as DEAD in the GCS. This is
+      // currently hard to do because the GCS first expects an ALIVE
+      // entry in the actor's log before we add the DEAD entry.
+      RAY_LOG(WARNING) << "Failed to restart actor " << actor_id;
```

Comment:
ERROR?

Reply:
Can we also log error for successful restarts? It should be a notable condition even if we recover from it, and ERROR is the only level that shows up on the console.